src/content/docs/basics/intercepting-requests.mdx

```diff
@@ -195,7 +195,20 @@ export const handlers = [
 
 ### How do I use relative URLs in Node.js?
 
-You don't. In Node.js, there is nothing to be relative to. If you are describing a network behavior for a Node.js application, use the absolute URLs you are requesting. If you are using MSW with a Node.js-based test runner, like Jest or Vitest, configure those runners accordingly to support relative URLs (i.e. polyfill the `window.location` object).
+You don't. In Node.js, there is nothing to be relative to. If you are describing a network behavior for a Node.js application, use the absolute URLs you are requesting. If you are using MSW with a Node.js-based test runner, like Jest or Vitest, configure those runners accordingly to support relative URLs (i.e. polyfill the `window.location` object), 
+
+### How to set `baseURL` in Node.js?
```

Comment:
Not sure if this belongs to the "Intercepting requests" section. I suppose it does. I need to think whether there's a better place to keep it (maybe the FAQ page?). What do you think?

Reply:
Yeah, you're spot on